CatanWizard.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE HTML>
 <html>
 <head>
-    <!-- <link rel="stylesheet" type="text/css" href="cw.css"/> -->
+    <link rel="stylesheet" type="text/css" href="main.css"/>
 </head>
 
 <body>
+    <h1>Generated Board Layout<br></h1>
     <p id="board">Before the script...</p>
 
+    <hr>
+    <form>
+
+        <form id="radios">Number of Players:
```

Comment:
Do we need the nested form for the radio buttons or can they exist as direct children of the outer form?